backend/common.js

```diff
@@ -103,7 +103,11 @@ export const makeHandleRequest = () => {
           'Response headers are potentially dangerous',
         );
 
-      res.writeHead(k8sResponse.statusCode, {
+      // change all 503 into 502
+      const statusCode =
+        k8sResponse.statusCode !== 503 ? k8sResponse.statusCode : 502;
```

Comment:
IMO it would be more readable without the negation 
https://moderatemisbehaviour.github.io/clean-code-smells-and-heuristics/general/g29-avoid-negative-conditionals.html
```suggestion
        k8sResponse.statusCode === 503 ? 502 : k8sResponse.statusCode ;
```

Reply:
My pleasure.